docs/tutorial/part-five/index.md

```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) before continuing here.
+> हे ट्यूटोरियल गॅट्सबीच्या डेटा लेयर बद्दलच्या मालिकेचा एक भाग आहे. आपण जात असल्याचे सुनिश्चित करा [part 4](/tutorial/part-four/) येथे सुरू ठेवण्यापूर्वी.
```

Comment:
गॅट्सबीच्या should be in English as it is !

Reply:
modified